xcode/Subconscious/Shared/Components/Common/Profile/FollowUserFormSheet.swift

```diff
@@ -324,9 +374,21 @@ struct FollowUserFormModel: ModelProtocol {
                 action: action,
                 environment: FormFieldEnvironment()
             )
+        case .failFollowDueToPetnameCollision(let error):
+            var model = state
+            model.failFollowMessage = error
```

Comment:
Is this error going to provide a user-friendly message?

If not, in the past, We've talked about defining a `LocalizedUserMessage` protocol, as kind of a user-facing complement to [`LocalizedError`](https://developer.apple.com/documentation/foundation/localizederror). If needed, this might be an opportunity to introduce?

Picturing:

```swift
protocol LocalizedUserMessage {
    var userMessage: String { get }
}
```

Reply:
Update: NVM, looks like we're using LocalizedError for this. That SGTM. Let's make it a convention that LocalizedError always provides a user-friendly description.